backend/analysis-engine/src/main/java/org/sonarsource/sonarlint/core/analysis/command/AnalyzeCommand.java

```diff
@@ -96,6 +95,8 @@ public AnalyzeCommand(String moduleKey, UUID analysisId, TriggerType triggerType
     this.files = files;
     this.extraProperties = extraProperties;
     this.futureResult = futureResult;
+
+    taskManager.createTask(analysisId, cancelMonitor);
```

Comment:
It works like this but it puzzles me to have a create method that doesn't return anything. Again it's probably a design issue around this tasks/commands